dev/test/Makefile

```diff
@@ -0,0 +1,164 @@
+CC ?= gcc
+# example: make test_dataloader TEST_CFLAGS=-fsanitize=address -fno-omit-frame-pointer 
+CFLAGS = -Ofast -Wno-unused-result -Wno-ignored-pragmas -Wno-unknown-attributes -g
+CFLAGS += $(TEST_CFLAGS)
+LDFLAGS =
+LDLIBS = -lm
+INCLUDES =
+CFLAGS_COND = -march=native
+
+# Find nvcc
+SHELL_UNAME = $(shell uname)
+REMOVE_FILES = rm -f
+OUTPUT_FILE = -o $@
+CUDA_OUTPUT_FILE = -o $@
+
+# NVCC flags
+# -t=0 is short for --threads, 0 = number of CPUs on the machine
+NVCC_FLAGS = -O3 -t=0 --use_fast_math -std=c++17
+NVCC_LDFLAGS = -lcublas -lcublasLt
+NVCC_INCLUDES =
+NVCC_LDLIBS =
+NCLL_INCUDES =
```

Comment:
nccl?

Reply:
Funny - that's in the Main makefile too. We should fix it there as well. And it's not being used. Noticed this the other day.

Reply:
Removed this.